src/test/java/nl/tudelft/pixelperfect/event/HostileShipTest.java

```diff
@@ -0,0 +1,16 @@
+package nl.tudelft.pixelperfect.event;
+
+/**
+ * Created by woute on 5/10/2016.
+ */
+public class HostileShipTest extends EventTest {
+
+  /**
+   * Factory method for testing.
+   * @return HostileShip class
+   */
+  public HostileShip createEvent() {
```

Comment:
No Override annotation here, as opposed to the factory method in for example the FireTest class?